src/components/Card/Card.tsx

```diff
@@ -7,7 +7,7 @@ import {
 import { makeStyles } from "@material-ui/styles";
 import _ from "lodash";
 import React, { useEffect, useRef } from "react";
-import badge from "../../assests/ispartner.svg";
+import badge from "../../assests/badge.svg";
```

Comment:
can you set a more meaningful name to this file ? because badge is quite generic and we do not understand jsut by the name of it that it a specific badge regarding beekeeper whom are partner with us. Try to think of a name that reflect this specificity, do not worry about the length of the name. The more important is that any dev with just a glimpse of the name, know perfectly what it is and which context it can be use